diff --git a/node_modules/chainmap/dist/index.d.ts b/node_modules/chainmap/dist/index.d.ts
index 11ae4ab..aab3482 100644
--- a/node_modules/chainmap/dist/index.d.ts
+++ b/node_modules/chainmap/dist/index.d.ts
@@ -1,4 +1,4 @@
-declare const _default: import("./types").Proxied<{
+declare const _default: import("./types.js").Proxied<{
     moonbeam: {
         beam: {
             explorerUrl: string;
@@ -414,6 +414,6 @@ declare const _default: import("./types").Proxied<{
             rpc: string[];
         };
     };
-}, import("./types").ChainInfo>;
+}, import("./types.js").ChainInfo>;
 export default _default;
-//# sourceMappingURL=index.d.ts.map
\ No newline at end of file
+//# sourceMappingURL=index.d.ts.map
